cmd/config/config.go

```diff
@@ -24,7 +24,20 @@ const (
 	UnsetFlag = "unset"
 )
 
-func NewConfigCmd(analyticsTracker analytics.Tracker) *cobra.Command {
+type ConfigCmd struct {
```

Comment:
We need to carry the state to know if we showed the help text, whether because the user passed `--help` or because they didn't call the command correctly. For non-resource commands, we need to create a struct to hold this state, along with the command itself.